web/app/pear/pear.module.css

```
@@ -0,0 +1,680 @@
+   ────────────────────────────────────────────────────────── */
+
+.page {
+  --pear: #6cb24a;
+  --pear-soft: #a6d388;
+  --pear-deep: #4f8f33;
+  /* Shared band background used by the home "Works with every agent" and
+     "Open source from day one" sections. */
+  --agent-tools-band-bg:
+    radial-gradient(circle at 18% 24%, rgba(116, 184, 226, 0.16), transparent 32%),
+    linear-gradient(180deg, #10273a 0%, #17364c 52%, #0d2638 100%);
+  min-height: 100vh;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)